usr/src/common/util/bzero.c

```diff
@@ -37,9 +37,14 @@
 #include "lint.h"
 #endif /* !_KMDB && !_BOOT */
 
+#if defined(_BOOT)
```

Comment:
_BOOT here is for dboot, not for loader, yes?  (I'm easily confused)

Reply:
Yes, this is a bit confusing. _STANDALONE refers to the `usr/src/boot` standalone while _BOOT indicates the `dboot` environment.